### PR TITLE
Run the community build CI with the correct sbt wrapper

### PR DESCRIPTION
**Pull request by smarter**
In particular, this avoids situations where the JVM goes into
interpreter-only mode because CodeCache is full.

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -41,7 +41,7 @@ pipeline:
       - cp -R . /tmp/3/ && cd /tmp/3/
       - git submodule update --init --recursive --jobs 7
       - export PATH=/tmp/4/project/scripts:$PATH
```
Comment (OlivierBlanvillain) [inline .drone.yml]: I thought this export would be enough for drone to pick `./project/scripts/sbt` over the default.

Comment (smarter) [reply .drone.yml]: Yeah I don't know why that doesn't work.

Comment (smarter) [reply .drone.yml]: Ah, it's the wrong /tmp folder

Comment (OlivierBlanvillain) [reply .drone.yml]: :man_facepalming:

**Pull request merged**